lib/spaces.js

```diff
@@ -40,78 +44,79 @@ function space_postgres_to_response(payload) {
 }
 
 // private
-const select_space = query.bind(query, fs.readFileSync('./sql/select_space.sql').toString('utf8'), (n) => { return n; });
-const insert_space = query.bind(query, fs.readFileSync('./sql/insert_space.sql').toString('utf8'), (n) => { return n; });
+const select_space = query.bind(query, fs.readFileSync('./sql/select_space.sql').toString('utf8'), (n) => n);
+const insert_space = query.bind(query, fs.readFileSync('./sql/insert_space.sql').toString('utf8'), (n) => n);
 const select_spaces = query.bind(query, fs.readFileSync('./sql/select_spaces.sql').toString('utf8'), space_postgres_to_response);
 
 // public
-async function list(pg_pool, req, res, regex) {
-  let data = await select_spaces(pg_pool, [])
-  return http_help.ok_response(res, JSON.stringify(data))
+async function list(pg_pool, req, res /* regex */) {
+  const data = await select_spaces(pg_pool, []);
+  return http_help.ok_response(res, JSON.stringify(data));
 }
 
 // public
 async function get(pg_pool, req, res, regex) {
-  let space_key = http_help.first_match(req.url, regex)
-  let space = await common.space_exists(pg_pool, space_key)
-  let data = await select_space(pg_pool, [space.name])
-  if(data.length === 0) {
-    throw new common.NotFoundError('The specified space does not exist.')
+  const space_key = http_help.first_match(req.url, regex);
+  const space = await common.space_exists(pg_pool, space_key);
+  const data = await select_space(pg_pool, [space.name]);
+  if (data.length === 0) {
+    throw new common.NotFoundError('The specified space does not exist.');
   }
   return http_help.ok_response(res, JSON.stringify(space_postgres_to_response(data[0])));
 }
 
 // public
-async function create(pg_pool, req, res, regex) {
-  let reserved_spaces = ["kube-system", "brokers", "k2-poc", "kube-public", "akkeris-system", "istio-system", "cert-manager"];
+async function create(pg_pool, req, res /* regex */) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function 'create' has a complexity of 23. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)